docs/managing-e2e-tests.md

```diff
@@ -14,22 +14,22 @@ The following folder contains all the SIG Cluster Lifecycle (the SIG that mainta
 Please note that this document will only cover details on the `kubeadm*.yaml` files.
 
 For example, let's have a look at this file:
-[kubeadm.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml)
+[kubeadm.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml)
 
 It contains a list of jobs such as:
 ```
-- name: ci-kubernetes-e2e-kubeadm-gce-master
+- name: ci-kubernetes-e2e-kubeadm-kind-master
   interval: 2h
   ...
 ```
 
-In this case, `ci-kubernetes-e2e-kubeadm-gce-master` is a test job that runs every 2 hours and it also has a set of parameters defined for it. This document will only cover some of them.
+In this case, `ci-kubernetes-e2e-kubeadm-kind-master` is a test job that runs every 2 hours and it also has a set of parameters defined for it. This document will only cover some of them.
 
 Important to note here is that testgrid jobs have a docker `image` that contains all the tools needed to run tests. Among them, one of the most relevant ones, is the tool that will be used for the deployment of the cluster.
 
-For this example job the `deployment` tool is called [kubernetes-anywhere](https://github.com/kubernetes/kubernetes-anywhere) (with integration in [this file](https://github.com/kubernetes/test-infra/blob/master/kubetest/anywhere.go)) and the `image` is called `gcr.io/k8s-testimages/e2e-kubeadm:...`. As a very high level summary, the way this works is when a job is invoked all the job parameters are passed to a CLI tool called `kubetest`, which then instantiates the job container and then the deployment tool inside it.
+For this example job the `deployment` tool is called [kind](https://github.com/kubernetes-sigs/kind) (with integration in [this file](https://github.com/kubernetes/test-infra/blob/master/kubetest/kind/kind.go)) and the `image` is called `*kubekins:...`. As a very high level summary, the way this works is when a job is invoked all the job parameters are passed to a CLI tool called `kubetest`, which then instantiates the job container and then the deployment tool inside it.
 
-Please note that there are plans for replacing kubernetes-anywhere with a new tool based on the [Cluster API](https://github.com/kubernetes-sigs/cluster-api).
+The SIG also uses another deployment tool called [kinder](https://github.com/kubernetes/kubeadm/kinder). kinder is based on kind and it's used for upgrades and version skew tests, but it does not require kubetest integration.
```

Comment:
Might be worth to add a link to configuration file for kinder jobs

Reply:
adding paragraph about workflow + example link.